superset/viz.py

```diff
@@ -439,14 +439,26 @@ def cache_key(self, query_obj: QueryObjectDict, **extra: Any) -> str:
 
     def get_payload(self, query_obj: Optional[QueryObjectDict] = None) -> VizPayload:
         """Returns a payload of metadata and data"""
+
         self.run_extra_queries()
         payload = self.get_df_payload(query_obj)
 
         df = payload.get("df")
+
+        # Check incompatible filters. Probably a better spot for this.
+        filters = self.form_data.get("filters")
+        labels = set()
+        for flt in filters:
+            labels.add(flt.get("col"))
+        # This doesn't seem to work for the date-time columns. Probably due to date time fields getting prefixed
```

Comment:
(Ville) Check if filters can handle temporal types correctly - applies to connectors/SQLA/models.py logic